pkg/installmanager/installmanager.go

```diff
@@ -432,6 +433,12 @@ func (m *InstallManager) Run() error {
 		m.log.WithError(err).Error("error reading cluster metadata")
 		return errors.Wrap(err, "error reading cluster metadata")
 	}
+	scrubbedMetadataBytes, err := scrubMetadataJSON(metadataBytes)
+	if err != nil {
+		m.log.WithError(err).Error("error cleaning up metadata")
+		scrubbedMetadataBytes = []byte("{}")
```

Comment:
Oh, I just noticed we're not actually erroring out here. I think we want to. That's the lesser evil vs. laying down an empty metadata.json, which will result in problems later in the flow from the methods that rely on it.